modules/adyoulikeBidAdapter.md

```diff
@@ -11,19 +11,49 @@ Banner formats are supported.
 
 # Test Parameters
 ```
-    var adUnits = [
-           {
-               code: 'test-div',
-               sizes: [[300, 250]],
-               bids: [
-                   {
-                       bidder: "adyoulike",
-                       params: {
-                           placement: 194f787b85c829fb8822cdaf1ae64435,
-                           DC: 'fra01', // Optional for set the data center name
-                       }
-                   }
-               ]
-           }
-       ];
+    var adUnits = {
+    "code": "test-div",
+    "mediaTypes": {
+      "banner": {
+        "sizes": ["300x250"]
+      },
+      "native": {
+        "image": {
+          "required": true,
+        },
+        "title": {
+          "required": true,
+          "len": 80
+        },
+        "cta": {
+          "required": false
+        },
+        "sponsoredBy": {
+          "required": true
+        },
+        "clickUrl": {
+          "required": true
+        },
+        "privacyIcon": {
+          "required": false
+        },
+        "privacyLink": {
+          "required": false
+        },
+        "body": {
+          "required": true
+        },
+        "icon": {
+          "required": true,
+          "sizes": []
+        }
+      }
+      bids: [{
+        bidder: "adyoulike",
+        params: {
+          placement: 194 f787b85c829fb8822cdaf1ae64435,
```

Comment:
Looks like an accidental space in the placement. Very minor, but it could throw some people off.

Reply:
Indeed, will be fixed in a future release. thx